tools/lsreg/README.md

```
@@ -0,0 +1,55 @@
+# lsreg
+
+Report and prune Launch Services registrations for `com.jwp23.throwntom`.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)